main/FatesIOVariableKindMod.F90

```diff
@@ -51,6 +51,17 @@ module FatesIOVariableKindMod
   character(*), parameter, public :: site_elcwd_r8 = 'SI_ELEMCWD_R8'
   character(*), parameter, public :: site_elage_r8 = 'SI_ELEMAGE_R8'
 
+
+  ! These are group indices for output variables. We use
+  ! these groups to do things like zero-ing and initializing
+
+  integer, parameter, public :: group_dyna_simple = 1
+  integer, parameter, public :: group_dyna_complx = 2
+  integer, parameter, public :: group_hifr_simple = 3
+  integer, parameter, public :: group_hifr_complx = 4
+  integer, parameter, public :: group_hydr_simple = 5
+  integer, parameter, public :: group_hydr_complx = 6
```

Comment:
Since these are passed to `set_history_var` via `upfreq`, the descriptive comment for `upfreq` in `fates_history_variable_type` definition probably should get an update.

Reply:
added text